src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -389,6 +393,36 @@ internal void Log(RpcEvent msg)
             }
         }
 
+        internal void SystemLog(RpcEvent msg)
+        {
+            RpcLog systemLog = msg.Message.RpcLog;
+            LogLevel logLevel = (LogLevel)systemLog.Level;
```

Comment:
this conversion might not be guaranteed. Add default switch to log as informational.